src/Foundation/Connect/tests/Managers/Account/AccountManagerV2Tests.cs

```diff
@@ -40,31 +43,174 @@ public class AccountManagerV2Tests
 
         private readonly IFixture fixture;
 
-        private readonly GetUserResult getUserResult;
-
-        private readonly ILogService<CommonLog> logService;
-
         public AccountManagerV2Tests()
         {
             var connectServiceProvider = Substitute.For<IConnectServiceProvider>();
+            var logService = Substitute.For<ILogService<CommonLog>>();
 
             this.customerServiceProvider = Substitute.For<CustomerServiceProvider>();
             connectServiceProvider.GetCustomerServiceProvider().Returns(this.customerServiceProvider);
+
+            this.accountManager = Substitute.For<AccountManagerV2>(connectServiceProvider, logService);
+
             this.fixture = new Fixture();
-            this.getUserResult = this.fixture.Build<GetUserResult>()
-                .With(res => res.Success, true)
-                .Create();
-            this.logService = Substitute.For<ILogService<CommonLog>>();
+        }
 
-            this.accountManager = Substitute.For<AccountManagerV2>(connectServiceProvider, this.logService);
+        public static IEnumerable<object[]> PartiesParameters =>
+            new List<object[]>
+            {
+                new object[] { null, Enumerable.Empty<Party>() },
+                new object[] { new CommerceCustomer(), null }
+            };
 
-            this.accountManager.Execute(Arg.Any<GetUserRequest>(), this.customerServiceProvider.GetUser)
-                .Returns(this.getUserResult);
+        #region GetUsers
+
+        [Fact]
+        public void GetUsers_ShouldCallExecuteMethod()
```

Comment:
Get users also:
1. Creates new model inside
2. Map parameters, checking whether they null or not (`??`)

Reply:
Simplified `GetUsers` method